tests/examples/ERC20_solidity_compatible/nonviper/ERC20_serpent.se

```diff
@@ -0,0 +1,100 @@
+# Author: Florian Tramer
```

Comment:
Why are we including a Serpent token when the Serpent compiler isn't safe for production?

Reply:
Only there to show compatibility in theory, but I will remove since you are right it's a bit pointless to have people install the dependency.